docs/quick-start/lua-howto.md

```diff
@@ -0,0 +1,95 @@
+
+For `Max Power`: the options are `10, 25, 50, 100, 250, 500, 1000 & 2000` mW. With Dynamic Power `off` this is the power level your TX transmits at. Selecting a power level higher that your TX supports will revert down to the max supported level.
+
+For `Dynamic`: the options are `Off, On, AUX9, AUX10, AUX11, AUX12`. `Off` means that the TX will transmit at Max Power constantly. `On` means the TX will dynamically lower power below max to preserve energy. `AUX9-12` means that the TX can be changed from max power to dynamic power by changing the position of the switch where switch ON = dynamic power, switch OFF = max power. For more information, [this](../../software/dynamic-transmit-power) provides a deeper dive on the algorithm and usage.
+
+### VTX Administrator
+<img src="../../assets/images/lua/vtxrm.jpg" width = "30%">
+
+This is the second folder in the LUA script and allows you to change your VTX settings directly from your radio. This requires some initial setup, but after that allows for quick VTX setting changes. Pressing it enters the folder, and if you want to exit the folder, press RTN.
+
+VTX Settings are sent every time a new TLM connection is acquired, or when `[Send VTX]` is pressed. **VTX Administrator can only send data when disarmed**
+
+For `Band`: the options are `Off, A, B, E, F, R & L`, which are the standard analog FPV bands. Off means that VTX Administrator will not send data.
+
+For `Channel`: the options are `1, 2, 3, 4, 5, 6, 7 & 8` which are the standard channels in the above bands.
+
+For `Pwr Lvl`: the options are `-, 1, 2, 3, 4, 5, 6, 7, & 8` which are the power levels that your VTX can do. Refer to your VTX table for the exact power levels - for example a vtx may have (1 = 25mW, 2 = 100mwW, 3 = 200mW, 4 = 500mW etc.).
+
+For `Pitmode`: the options are `On & Off` which allow you to quickly switch into or out of pitmode
+
+Finally pressing the `[Send VTX]` button sends your configuration and changes the VTXes settings.
+
+### Bind
+<img src="../../assets/images/lua/bindrm.jpg" width = "30%">
+
+Pressing the `[Bind]` button activates binding mode for traditional binding. For  more information check out this page on [binding](./../binding).
+
+### Wifi Update (ESP32 TXes Only)
+<img src="../../assets/images/lua/wifirm.jpg" width = "30%">
+
+Pressing the `[Wifi Update]` selection activates Wifi Update mode on ESP32 TXes. Visit [this page](../../software/updating/wifi-updating.md) for instructions on how the updating process works. Reboot or change models to exit this mode.
+
+### BLE Joystick (ESP32 TXes Only)
+<img src="../../assets/images/lua/blerm.jpg" width = "30%">
+
+Pressing the `[BLE Joystick]` selection activates Bluetooth mode which allows connection to most flight simulators through the bluetooth of your computer. Reboot or change models to exit this mode.
```

Comment:
I would add more information that this 2 buttons require a 2nd press of "OK" to proceed after the POP UP screen asked the USER to press "OK" to continue. this feature is to prevent user accidentally pressing on it and get stuck in wifi mode or BLE Joystick mode.

Reply:
I feel like this doesn't really need specification as users should be able to understand one button. please.